.changesets/feat_feat_local_safelist.md

```diff
@@ -0,0 +1,17 @@
+### Allow use of a local persisted query manifest for use with offline licenses ([Issue #4587](https://github.com/apollographql/router/issues/4587))
```

Comment:
filename `feat_feat_`?

Reply:
My branch was titled `feat/local-safelist` and the title is based off the branch name, so, yeah. I can prune it though, good catch.